src/lib/progress-spinner/progress-spinner.scss

```diff
@@ -33,8 +33,8 @@ $mat-progress-spinner-viewport-size: 100px !default;
   path {
     fill: transparent;
 
-    // Stroke width of 10px defines stroke as 10% of the viewBox.
-    stroke-width: $mat-progress-spinner-stroke-width;
+    // Stroke width is by default 10px, which is relatively to the viewBox size 10%.
```

Comment:
It currently is centered properly in the viewBox in the `getSvgArc` method, we define `pathRadius` as 40 and `radius` as 50.  If we allow the `strokeWidth` to be set we will need to make a corresponding adjustment to the pathRadius.  

Thought we need to keep in mind  `getSvgArc` is called ~60 times/sec, so ideally we don't want to recalculate the `pathRadius` value every time if we can avoid it, though it is a pretty trivial calculation compared to determining easing values.